unittests/conftest.py

```diff
@@ -0,0 +1,7 @@
+import sys
```

Comment:
can we move this under sspl-test itself

Reply:
pytest will look into the unittest directories for conftest.py. (right now it is unittests)

Reply:
Shouldn't we have a single directory for all our tests? I mean there is no restriction, just a folder structure convention.
Also can we specify location of `conftest.py` in `pytest.ini`

Reply:
moved conftest.py to unittests directory where all unittests will be. yes, all tests will be there in unittests dir.

Reply:
Okay